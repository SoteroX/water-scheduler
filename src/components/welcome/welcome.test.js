import React from "react";
import { shallow } from "enzyme";
import { render } from "@testing-library/react";
// Import components
import Welcome from "./welcome.component";

describe("Welcome component", () => {
  // Makes sure the Header section in Welcome component is render
  test("renders Welcome header", () => {
    const { getByText } = render(<Welcome />);
    expect(
      getByText("Welcome to Tandem Watering Scheduler")
    ).toBeInTheDocument();
  });
  // Makes sure the Paragraph section in Welcome component is render
  test("renders Welcome paragraph", () => {
    const { getByText } = render(<Welcome />);
    expect(
      getByText(
        "Tandem Watering Scheduler is an application that helps you keep track of when your plants need watering"
      )
    ).toBeInTheDocument();
  });
  // Makes sure Welcome component is render
  it("renders App component without crashing", () => {
    expect(shallow(<Welcome />)).toMatchSnapshot();
  });
});
