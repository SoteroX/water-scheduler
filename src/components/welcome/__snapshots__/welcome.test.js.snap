// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component renders App component without crashing 1`] = `
<div
  className="welcome-container"
>
  <h1>
    Welcome to Tandem Watering Scheduler
  </h1>
  <p>
    Tandem Watering Scheduler is an application that helps you keep track of when your plants need watering
  </p>
</div>
`;
